submissions/mod-comparison/mod-comparison.md

```diff
@@ -0,0 +1,107 @@
+## Factorio Mod Comparison: <author>OneCruelBagel</author>
+
+A lot of the appeal of Factorio for me comes from the many different ways it can be approached – you can concentrate on launching rockets as quickly as possible or you can work on secure defences to hold back ever increasing waves of biters. You could build your base around a gigantic maze of belt spaghetti or maybe you prefer to run clouds upon clouds of bots. You can create a world with massive, dense ore patches to allow you to concentrate on building, or you can set up endless train networks to bring in resources from far across the planet.
```

Comment:
```suggestion
A lot of the appeal of Factorio for me comes from the many different ways it can be approached – you can concentrate on launching rockets as quickly as possible, or you can work on secure defenses to hold back ever-increasing waves of biters. You could build your base around a gigantic maze of belt spaghetti or maybe you prefer to run clouds upon clouds of bots. You can create a world with massive, dense ore patches to allow you to concentrate on building, or you can set up endless train networks to bring in resources from far across the planet.
```